decidim-proposals/config/locales/en.yml

```diff
@@ -53,6 +53,7 @@ en:
             proposal_edit_before_minutes: Proposals can be edited by authors before this many minutes passes
             proposal_limit: Proposal limit per user
             vote_limit: Vote limit per user
+            proposal_length: Maximum proposal body length
```

Comment:
Tests are failing because this key is not sorted alphabetically, can you fix it please?